csrc/codegen.cpp

```diff
@@ -1071,7 +1071,23 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
     std::stringstream ss;
     auto macro = mma->macro();
     ss << genArchString(macro) << "::";
-    ss << toString(macro);
+
+    switch (macro) {
```

Comment:
I understand that it makes more sense to keep this as a separate function, but this entire `genMmaOp` will be removed soon, so I don't really care about code quality.